src/vm/excep.cpp

```diff
@@ -5152,10 +5138,20 @@ LONG InternalUnhandledExceptionFilter(
 
 } // LONG InternalUnhandledExceptionFilter()
 
+static bool useEntryPointFilter = false;
+
+void ParseUseEntryPointFilter(LPCWSTR value)
+{
+#ifndef PLATFORM_WINDOWS // This feature has only been tested on Windows, keep it disabled on other platforms
```

Comment:
Should this be `#ifdef PLATFORM_WINDOWS` ?

Reply:
Mornings